packages/core/src/components/RadioButton/RadioButton.tsx

```diff
@@ -24,10 +25,10 @@ const RadioButton: React.FC<RadioButtonProps> = ({
   Icon,
   disabled = false,
   color,
-  value,
+  value = "",
   selected,
   unselectedColor,
-  onPress = () => {},
+  onPress,
```

Comment:
any reason you've removed the default function? I think it was there for a reason, unless thats part of the bug fix

Reply:
@peter Unneeded. If one exists it will run.  See line 45.